tutorials/cbt-import-csv.md

```diff
@@ -96,18 +96,15 @@ code:
 ### Start the Dataflow job 
 
     mvn package exec:exec -DCsvImport -Dbigtable.projectID=YOUR_PROJECT_ID -Dbigtable.instanceID=YOUR_INSTANCE_ID
-    -DinputFile="YOUR_FILE" -Dheaders="YOUR_HEADERS"
+    -DinputFile="YOUR_FILE" -Dbigtable.table=YOUR_TABLE_ID -Dheaders="YOUR_HEADERS"
```

Comment:
At first glance I think `YOUR_TABLE_ID` should be in double quotes.

Reply:
Whoops! Good catch